django/db/migrations/operations/utils.py

```diff
@@ -19,7 +19,8 @@ def resolve_relation(model, app_label=None, model_name=None):
                 )
             return app_label, model_name
         if '.' in model:
-            return tuple(model.lower().split('.', 1))
+            splitted = model.split('.', 1)
```

Comment:
```suggestion
            app_label, model_name = model.split('.')
```

Reply:
That's better indeed, changed this in 0969b23